packages/sn-universal-header-react/src/components/hamburger-menu.tsx

```diff
@@ -89,7 +90,13 @@ export const HamburgerMenu: React.FunctionComponent<HamburgerMenuProps> = (props
         <Paper className={classes.popperWrapper}>
           <ClickAwayListener onClickAway={handleClose}>
             <MenuList autoFocusItem={isPopperOpen}>
-              <Link href="https://github.com/SenseNet/sn-client/tree/master/examples/sn-react-tasklist" target="_blank">
+              <Link
+                href={
+                  props.appName
+                    ? `https://github.com/SenseNet/sn-client/tree/master/examples/${props.appName}`
+                    : 'https://github.com/SenseNet/sn-client/tree/master/examples/sn-react-tasklist'
```

Comment:
https://github.com/SenseNet/sn-client/tree/master/examples would be a better default value